index.d.ts

```diff
@@ -0,0 +1,9 @@
+type fn = (str: string) => string;
+
+interface kebabCase extends fn {
+    reverse: fn
+}
+
+const kebabCase: kebabCase;
+
+export default kebabCase;
```

Comment:
IMO types should be PascalCased/UpperCamelCased, i.e. I suggest changing this to:
```suggestion
type Fn = (str: string) => string;

interface KebabCase extends Fn {
    reverse: Fn
}

const kebabCase: KebabCase;

export default kebabCase;
```